pkg/provider/oracle/oracle.go

```diff
@@ -71,13 +71,14 @@ var _ esv1.Provider = &VaultManagementService{}
 
 // VaultManagementService implements the External Secrets provider interface for Oracle Cloud Infrastructure Vault.
 type VaultManagementService struct {
-	Client                VMInterface
-	KmsVaultClient        KmsVCInterface
-	VaultClient           VaultInterface
-	vault                 string
-	compartment           string
-	encryptionKey         string
-	workloadIdentityMutex sync.Mutex
+	Client                  VMInterface
+	KmsVaultClient          KmsVCInterface
+	VaultClient             VaultInterface
+	vault                   string
+	compartment             string
+	encryptionKey           string
+	workloadIdentityMutex   sync.Mutex
+	authConfigurationsCache map[string]auth.ConfigurationProviderWithClaimAccess
```

Comment:
It appears this map will grow in size over the lifetime of the operator. Perhaps not enough to make a difference, but "old" store resource versions will accumulate clients over time.